cluster/saltbase/salt/kube-proxy/kube-proxy.manifest

```diff
@@ -59,7 +59,7 @@ spec:
     command:
     - /bin/sh
     - -c
-    - kube-proxy {{api_servers_with_port}} {{kubeconfig}} {{cluster_cidr}} --resource-container="" {{params}} 1>>/var/log/kube-proxy.log 2>&1
+    - echo -998 > /proc/$$$/oom_score_adj && kube-proxy {{api_servers_with_port}} {{kubeconfig}} {{cluster_cidr}} --resource-container="" {{params}} 1>>/var/log/kube-proxy.log 2>&1
```

Comment:
I just tried this. No sure why it failed for me. `$$` works for me tough. But e2e is not complaining. 

```
$ cat  /proc/$$$/oom_score_adj
cat: /proc/29756$/oom_score: No such file or directory
```